services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImplV2.java

```diff
@@ -100,7 +110,7 @@ public DataPlaneClientImplV2(Config globalConfig, ArrayList<String> monitorHosts
         LOG.info("Done printing out all monitorHosts");
         this.numberOfGrpcChannelPerHost = globalConfig.numberOfGrpcChannelPerHost;
         this.numberOfWarmupsPerChannel = globalConfig.numberOfWarmupsPerChannel;
-        this.hostAgentPort = 50001;
+        this.hostAgentPort = 9016;
```

Comment:
If ACA port is changed to to 9016, we would need to change ACA build/run script as well, right?

Reply:
@zzxgzgz @DavidLiu506 for both of you to review.

Reply:
I think you don't have to change the build script, you just need to specify the ACA port number via the `-s` flag when starting ACA. If you don't specify ACA's port number, ACA will use its default port, which is 50001.

Reply:
This is NCM grpc port not ACA port.

Reply:
NCM using 9016 port for grpc and ACA using 50001 for grpc.